charts/cluster-autoscaler/values.yaml

```diff
@@ -358,8 +358,8 @@ prometheusRule:
   additionalLabels: {}
   # prometheusRule.namespace -- Namespace which Prometheus is running in.
   namespace: monitoring
-  # prometheusRule.interval -- How often rules in the group are evaluated (falls back to `global.evaluation_interval` if not set).
-  interval: null
+  # prometheusRule.interval -- How often rules in the group are evaluated (falls back to the Prometheus level value of `global.evaluation_interval` if not set).
+  interval: nil
```

Comment:
I don't think this change is necessary, templating with this as the default value:
```yaml
spec:
  groups:
    - name: release-name-aws-cluster-autoscaler
      interval: nil
      rules:
        []
```

Templating with the new logic, but the old `interval: null`:
```yaml
spec:
  groups:
    - name: release-name-aws-cluster-autoscaler
      rules:
        []
```